app/models/venue.rb

```diff
@@ -1,2 +1,9 @@
 class Venue < ActiveRecord::Base
+  def display_t_accessible
```

Comment:
While I get the intent of this method - and it 100% works - , consider looking into "Facade" patterns in the future for a better place to hold this logic.